tests/run-make/README.md

```diff
@@ -0,0 +1,35 @@
+# The `run-make` test suite
+
+The `run-make` test suite contains tests which are the most flexible out of all
+the [rust-lang/rust](https://github.com/rust-lang/rust) test suites. `run-make`
+tests can basically contain arbitrary code, and are supported by the
+[`run_make_support`] library.
+
+## Infrastructure
+
+The implementation for collecting and building the `rmake.rs` recipes (or
```

Comment:
Maybe one sentence that describes that there are two types of runmake tests would be useful.

Reply:
Explained that there are two kinds of run-make tests.